server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -58,7 +58,8 @@ data:
             "endpoint": "{{ .Values.zookeeper.url }}"
         },
         "storage": {
-            "enableWholeSummaryUpload": {{ .Values.storage.enableWholeSummaryUpload }}
+            "enableWholeSummaryUpload": {{ .Values.storage.enableWholeSummaryUpload }},
+            "storageUrl": "{{ .Values.gitrest.internalUrl }}",
```

Comment:
Can we have
```
"storageUrl": "{{ .Values.storage.storageUrl }}",
```
or similar? The configmap now creates the storageUrl in the storage section, but in values.yaml it still points to the gitrest section

Reply:
And we also need to add that new value to values.yaml - sorry if I missed that, but I think we haven't done it yet in this PR:
https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/kubernetes/routerlicious/values.yaml

Reply:
Yup. Done. :)